hawc/apps/animal/migrations/0031_auto_20210216_1238.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.3 on 2021-02-16 17:38
```

Comment:
instead of a new migration, can you just add `blank=True,` to the django31 migration. Since this doesn't affect the db I try to remove these  IF done correctly, running makemigrations should be a noop.